templates/hound.yml

```diff
@@ -0,0 +1,11 @@
+# See https://houndci.com/configuration for configuration details.
+coffeescript:
+  enabled: true
+haml:
+  enabled: true
+javascript:
+  enabled: false
+ruby:
+  enabled: true
+scss:
+  enabled: true
```

Comment:
Since these are enabled in Hound by default, is the idea here to be explicit anyway?